library_conventions/packages.adoc

```diff
@@ -193,6 +193,66 @@ image:img/footprint_pin1_sot669.png[]
 ====
 
 
+[#libraryconventions-packages-placement]
+==== Placement Layer
+
+The _Top Placement_ layer is intended to be printed on silkscreen and thus
+should contain information required for assembling the PCB. But don't put too
+many things on that layer as it would waste space on the PCB!
+
+Typically this layer should only contain some lines and dots to indicate where
+and in which orientation the device gets assembled, for example an outline and
+a dot next to pin 1.
+
+*The placement should be drawn according {ipc7351c-slides}[IPC-7351C]*. The
+most important rules are the following:
+
+* *It must stay visible after assembling the package* to allow reviewing
+  positioning and orientation of assembled devices. In other words, the
+  placement layer must only contain drawings _around_ the package's body,
+  but not _under_ it!
```

Comment:
I'd relax those rules to "should" instead of "must". Maybe I'd also change "only" to "primarily".

For example for the DIP packages, we put a "pin 1 indicator dot" *on* the package body. This would not be allowed according to the current wording (even if it's in addition to an IPC pin 1 indicator).

Reply:
Done.